src/components/Modal/FreeBoardModal.tsx

```diff
@@ -0,0 +1,76 @@
+import { SubmitHandler, useForm } from 'react-hook-form';
+import profile from '../../../public/profile.svg';
+import TextButton from '@/components/common/TextButton';
+import ModalLabel from '@/components/common/modal/ModalLabel';
+import ModalLayout from '@/components/common/modal/ModalLayout';
+import { useAxios } from '@/hooks/useAxios';
+
+type Inputs = {
+  title: string;
+  content: string;
+};
+
+interface FreeBoardModalProps {
+  closeClick: () => void;
+}
+// POST axios 형식
+// {
+//   "title": "string", //  요게 없음 ui에
+//   "content": "string",
+//   "isAnnouncement": false // 댓글 기능은 나중에?
+// }
+
+export default function FreeBoardModal({ closeClick }: FreeBoardModalProps) {
+  const { fetchData } = useAxios({});
+  const { register, watch, handleSubmit } = useForm<Inputs>();
+  const onSubmit: SubmitHandler<Inputs> = (data) => {
+    const createFreeBoard = {
+      content: data.content,
+      title: data.title,
+    };
```

Comment:
```suggestion
  const onSubmit: SubmitHandler<Inputs> = ({ content, title }) => {
    const createFreeBoard = {
      content,
      title,
    };
```
이렇게 쓰면 더 깔끔해질 것 같아요!